__mocks__/@react-native-async-storage/async-storage.js

```diff
@@ -0,0 +1 @@
+export * from '@react-native-async-storage/async-storage/jest/async-storage-mock';
```

Comment:
To enable async storage for jest

I followed the instructions here: https://react-native-async-storage.github.io/async-storage/docs/advanced/jest/